src/ripple/protocol/SField.h

```diff
@@ -515,6 +515,7 @@ extern SF_VL const sfCreateCode;
 extern SF_VL const sfMemoType;
 extern SF_VL const sfMemoData;
 extern SF_VL const sfMemoFormat;
+extern SF_VL const sfData;
```

Comment:
Will we get any performance benefits if we use a custom `SF_256Byte` field instead of `SF_VL` (variable length) here?

I'm thinking of something like:
```
using SF_256Byte = TypedField<STBitString<256*8>>
```

Reply:
This is such a simple amendment, I don't think any performance benefits from using `SF_UINT256` would matter.  Also, such a data type would mean that the length of the data type _had_ to be 256 bytes (instead of possibly being less), which sure you could pad with 0s, but that's a little less nice of a DX.

Reply:
hmm okay, got your point. what is DX?

Reply:
Developer experience